analyzers/src/SonarAnalyzer.Common/Helpers/SonarAnalysisContext.cs

```diff
@@ -94,12 +94,14 @@ internal SonarAnalysisContext(AnalysisContext context, IEnumerable<DiagnosticDes
         public bool ShouldAnalyzeGenerated(Compilation c, AnalyzerOptions options) =>
             ShouldAnalyzeGenerated(context.TryGetValue, c, options);
 
-        public static bool ShouldAnalyze(TryGetValueDelegate<bool> tryGetValue,
+        public static bool ShouldAnalyze(TryGetValueDelegate<bool> tryGetValueBool,
```

Comment:
I would rename this to `tryGetBool` or `tryGetIsGeneratedCode`.

Reply:
The 'generated` thing is how we use it. But the delegate is general and could give us any bool value.

Reply:
Then `tryGetBool`? `Value` here does not help.